README.md

```diff
@@ -1,3 +1,5 @@
+> **_NOTE:_**  To use seed your local db, instal missing dependencies first with yarn, then do npx prisma migrate dev so the db tables would update to the current schema and prisma will auto run the seed script
+
```

Comment:
Can we note here the dependencies we have to install here? I all the ones I have to do.